### PR TITLE
[Visual] Achievement icons

### PR DESCRIPTION
**Pull request by frutescens**
## What are the changes?
This PR is an abridged, achievement-only version of this PR: https://github.com/pagefaultgames/pokerogue/pull/2149
It adds new ribbons, created by koda-want-to-sleep,  for the MonoGen achievements and brings Pokerogue's wording in line with official wording. The ribbons are based on ribbons within the games. Generation II's achievement uses the ribbon for defeating Red on Mt. Silver + Gen V's achievement uses the Royal Ribbon since the plot line of the B/W games was driven partly by royal intrigue. 

## Why am I doing these changes?
I just think it would be nice. 

## What did change?
New icons added.
Readme.md edited to give artist credit.
locales/x/achvs.ts edited for better wording 

### Screenshots/Videos
image

## How to test the changes?
Open Achievements window

## Checklist
- [ x] There is no overlap with another PR?
- [ x] The PR is self-contained and cannot be split into smaller PRs?
- [x ] Have I provided a clear explanation of the changes?
- [ x] Have I tested the changes (manually)?
    - [x ] Are all unit tests still passing? (`npm run test`)
- [ x] Are the changes visual?
  - [ x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/system/achv.ts [hunk 1]
```diff
@@ -323,15 +323,15 @@ export const achvs = {
...
+  MONO_GEN_FOUR_VICTORY: new ChallengeAchv("MONO_GEN_FOUR","",  "MONO_GEN_FOUR.description", "ribbon-gen4", 100, c => c instanceof SingleGenerationChallenge && c.value === 4),
+  MONO_GEN_FIVE_VICTORY: new ChallengeAchv("MONO_GEN_FIVE","",  "MONO_GEN_FIVE.description", "ribbon-gen5", 100, c => c instanceof SingleGenerationChallenge && c.value === 5),
+  MONO_GEN_SIX_VICTORY: new ChallengeAchv("MONO_GEN_SIX","",  "MONO_GEN_SIX.description", "ribbon-gen6", 100, c => c instanceof SingleGenerationChallenge && c.value === 6),
+  MONO_GEN_SEVEN_VICTORY: new ChallengeAchv("MONO_GEN_SEVEN","",  "MONO_GEN_SEVEN.description", "ribbon-gen7", 100, c => c instanceof SingleGenerationChallenge && c.value === 7),
+  MONO_GEN_EIGHT_VICTORY: new ChallengeAchv("MONO_GEN_EIGHT","",  "MONO_GEN_EIGHT.description", "ribbon-gen8", 100, c => c instanceof SingleGenerationChallenge && c.value === 8),
+  MONO_GEN_NINE_VICTORY: new ChallengeAchv("MONO_GEN_NINE","",  "MONO_GEN_NINE.description", "ribbon-gen9", 100, c => c instanceof SingleGenerationChallenge && c.value === 9),
```
Comment (torranx) [inline src/system/achv.ts:334]: please follow current naming convention (using `_` to separate words)

File: src/locales/en/achv.ts [hunk 2]
```diff
@@ -172,39 +172,39 @@ export const PGMachv: AchievementTranslationEntries = {
 
   "MONO_GEN_ONE": {
     name: "The Original Rival",
-    description: "Complete the generation one only challenge.",
+    description: "Complete the Generation I only challenge.",
```
Comment (torranx) [inline src/locales/en/achv.ts:175]: please change other locale files as well

**Comment by torranx** [comment]
please fix conflicts, thanks!

**Comment by frutescens** [comment]
hi! i'm a bit tied up right now but i promise i'll fix them by the weekend

**Comment by frutescens** [comment]
Hi, apologies for the delay. I just updated the file names + localizations.

**Comment by torranx** [comment]
> Hi, apologies for the delay. I just updated the file names + localizations. 

Please delete unused files as well ('ribbon-gen{num}')

**Comment by frutescens** [comment]
Done! Sorry, clueless with git

**Comment by torranx** [comment]
> Done! Sorry, clueless with git 

looks better, next would be to resolve the merge conflicts

**Comment by frutescens** [comment]
i can't resolve the conflicts 
image

**Comment by frutescens** [comment]
@Madmadness65 - I was wondering if you could help me resolve some of these conflicts with TexturePacker. My trial ran out.

**Comment by Tempo-anon** [comment]
I'd say to nix the achievement locale changes for now since they have merge conflicts and just bloat the PR. Just update the readme to credit koda_want_to_sleep, update the items.json and items.png (so they don't conflict with newly added stuff), and then just make achv.ts point to the new ribbons

**Pull request merged**